directory/directory.json

```diff
@@ -228,15 +228,17 @@
+
+	"charnatoken.top": {
```

Comment:
This is autogenerated by ./checkBuild.sh. You shouldn't be manually adding this

Reply:
okay . so i only have to change website to https and then maunally edit only on these files
directory.js directory.json  logos.json